Menu/MenuBuilder.php

```diff
@@ -119,7 +119,7 @@ public function createSidebarMenu()
                     }
 
                     $label = $admin->getLabel();
-                    $options = $admin->generateMenuUrl('list');
+                    $options = $admin->generateMenuUrl('list', $item['route_params'], $item['route_absolute']);
```

Comment:
In which case would `$item['route_params']` be necessary. It wasn't there before and apparently, nobody complained…